src/app/static/src/app/components/downloadResults/DownloadResults.vue

```diff
@@ -42,6 +42,18 @@
                         <tick color="#e31837" v-if="uploadComplete" width="20px"></tick>
                     </div>
                 </div>
+                <div id="releaseCreated" v-if="releaseCreated || releaseNotCreated" class="d-flex align-items-end">
+                    <div class="d-flex align-items-center height-40 mr-1">
+                        <span class="font-weight-bold" v-translate="releaseCreated ? 'releaseCreated' : 'releaseNotCreated'"></span>
+                    </div>
+                    <div class="d-flex align-items-center height-40">
+                        <tick color="#e31837" v-if="releaseCreated" width="20px"></tick>
+                        <svg class="tick" id="cross" v-if="releaseNotCreated" width="20px">
+                            <path d="M 10,10 l 90,90 M 100,10 l -90,90" stroke="#e31837" stroke-width="15"
+                                fill="none"></path>
+                        </svg>
```

Comment:
I could put this in its own component, like tick, if you think that would be valuable.